typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ytdl-core",
+  "main": "./lib/index.d.ts",
+  "dependencies": {},
+  "globalDependencies": {
+    "node": "registry:dt/node#6.0.0+20160613154055"
```

Comment:
Is this required, this module should work with node versions specified in `.travis-ci`. With typings, would it only work in nodev6? And why point to a release this specifically?